getting_started/build-guide/build-with-zephyr.rst

```diff
@@ -0,0 +1,69 @@
+.. _build-with-zephyr:
+
+Build SOF with Zephyr
+#####################
+
+.. contents::
+   :local:
+   :depth: 3
+
+This guide describes the process of building and running |SOF| as a Zephyr
+application.
```

Comment:
I would add a note here that it needs v1.6 onwards and is limited to some SOF platforms atm like CAVS.

Reply:
@lgirdwood Sure, can add, but in fact west will make sure to check out a suitable SOF version, and if you choose to override that, well, you better know what you're doing :-)